src/listmode_buildblock/LmToProjData.cxx

```diff
@@ -328,8 +328,13 @@ post_processing()
 
   // handle time frame definitions etc
   // If num_events_to_store == 0 && frame_definition_filename.size == 0
-  if(num_events_to_store==0 && frame_definition_filename.size() == 0)
-      do_time_frame = true;
+  do_time_frame = num_events_to_store<=0;
+
+  if (do_time_frame && frame_definition_filename.size()==0)
+  {
+      warning("Have to specify either 'frame_definition_filename' or 'num_events_to_store'\n");
```

Comment:
delete final `\n`. `warning` will add one. (I know there's still lots of them around with a trailing newline..)